Firebase/Core/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
-# v6.0.1 -- M48
+# v6.1.0 -- M52
+- [changed] `FIROptions.appGroupID` introduced. (#3293)
```

Comment:
s/changed/added/

Is there a phrase to add that would summarize what `FIROptions.appGroupID` does?

Reply:
Updated.

Reply:
Did you see the question I asked?

Reply:
Ah, sorry, I missed it.